m0kv_metadata_parsing_tool/Makefile

```diff
@@ -0,0 +1,5 @@
+COMPILER=g++
+all:
+	$(COMPILER) base64_encoder_decoder.cc -o base64_encoder_decoder -std=c++11
```

Comment:
use -Wall flag to catch any compiler warnings.

Reply:
Hi Amit, sure will do that.